AK/NeverDestroyed.h

```diff
@@ -35,7 +36,7 @@ class NeverDestroyed {
     const T& get() const { return reinterpret_cast<T&>(storage); }
 
 private:
-    alignas(T) u8 storage[sizeof(T)];
+    AK::aligned_storage<sizeof(T), alignof(T)>::type storage;
```

Comment:
Could maybe invent something like `TypeAlignedStorage<T>` or `SimpleAlignedStorage<T>` for the common cases of just hosting a single type?